Source/Plugins/Core/com.equella.core/js/tsrc/fileuploader/UploadFileName.tsx

```diff
@@ -27,16 +27,28 @@ interface UploadFileNameProps {
    * The link to preview the file
    */
   link?: string;
+  /**
+   * True if an indentation is required
```

Comment:
```suggestion
   * `true` if the entry should be indented.
```

Note that (unlike JavaDoc) JSDoc supported markdown, so best to use it when appropriate. (One day we may even generate the docs. :crossed_fingers: )

Reply:
Good idea!! Thank you!